docs/README.md

```diff
@@ -35,7 +47,7 @@ const css = `
 `
 ```
 
-## Class component example
+### Example without hooks
```

Comment:
Why did you change this?

I originally had this header say "Example without hooks," but I changed it to "Class component example" because I think that for a user of kremling the question is whether they can use this with their class components.

I don't think the header will make a big difference either way -- I'm just curious why you changed it.

Reply:
I think hooks are an intimidating topic while classes are more foundational. I named it this so users don't feel obligated to use hooks with Kremling.